e2e/pod.go

```diff
@@ -210,3 +210,25 @@ func deletePodWithLabel(label, ns string, skipNotFound bool) error {
 	}
 	return err
 }
+
+// getMountPath fetches the path on an app where volume is mounted.
+func getMountPath(app *v1.Pod) string {
```

Comment:
this won't be a generic function as you are always getting the first container first mountpoint.